content/ko/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md

```diff
@@ -85,3 +115,96 @@ scheduler:
       readOnly: true
       pathType: "File"
 ```
+
+### Etcd 플래그
+
+자세한 사항은 [etcd 서버 문서](https://etcd.io/docs/)를 확인한다.
+
+사용 예시:
+
+```yaml
+apiVersion: kubeadm.k8s.io/v1beta3
+kind: ClusterConfiguration
+etcd:
+  local:
+    extraArgs:
+      election-timeout: 1000
+```
+
+## 패치를 통해 컨트롤 플레인 사용자 정의하기 {#patches}
+
+{{< feature-state for_k8s_version="v1.22" state="beta" >}}
+
+Kubeadm을 사용하면 패치 파일이 있는 디렉토리를 개별 노드에 대한 `InitConfiguration`과 `JoinConfiguration`에
+전달할 수 있다. 이 패치는 컨트롤 플레인 컴포넌트 메니패스트가 디스크에 기록되기 전에
+최종 사용자 정의 단계로 사용될 수 있다.
+
+`--config <YOUR CONFIG YAML>`을 사용하여 이 파일을 `kubeadm init`에 전달할 수 있다.
+
+```yaml
+apiVersion: kubeadm.k8s.io/v1beta3
+kind: InitConfiguration
+nodeRegistration:
+  patches:
+    directory: /home/user/somedir
+```
+
+{{< note >}}
+`kubeadm init`의 경우, `---`로 구분된 `ClusterConfiguration`과 `InitConfiguration`을 모두
+포함하는 파일을 전달할 수 있다.
+{{< /note >}}
+
+`--config <YOUR CONFIG YAML>`을 사용하여 이 파일을 `kubeadm join`에 전달할 수 있다.
+
+```yaml
+apiVersion: kubeadm.k8s.io/v1beta3
+kind: JoinConfiguration
+nodeRegistration:
+  patches:
+    directory: /home/user/somedir
+```
+
+디렉토리는 `target[suffix][+patchtype].extension` 형태의 파일을 포함해야 한다.
+예를 들면, `kube-apiserver0+merge.yaml` 또는 단순히 `etcd.json`의 형태이다.
+
+- `target`은 `kube-apiserver`, `kube-controller-manager`, `kube-scheduler` 그리고 `etcd` 중 하나가 될 수 있다.
+- `patchtype`은 `strategic`, `merge` 그리고 `json` 중 하나가 될 수 있으며
+[kubectl에서 지원하는](/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch) 패치 형식을 준수해야 한다.
+`patchtype`의 기본값은 `strategic`이다.
+- `extension`은 `json` 또는 `yaml` 중 하나여야 한다.
+- `suffix`는 어떤 패치가 먼저 적용되는지를 결정하는 데 사용할 수 있는 영숫자 형태의
+선택적 문자열이다.
+
+{{< note >}}
+`kubeadm upgrade`를 사용하여 kubeadm 노드를 업그레이드하는 경우 동일한 패치를 다시
+제공해야 한다. 따라서, 업그레이드 이후에도 사용자 정의를 유지할 수 있다. 이는 동일한 디렉토리로 지정된 `--patches`
```

Comment:
[원문]

> If you are using kubeadm upgrade to upgrade your kubeadm nodes you must again provide the same patches, so that the customization is preserved after upgrade.

```suggestion
`kubeadm upgrade`를 사용하여 kubeadm 노드를 업그레이드하는 경우, 업그레이드 이후에도 
사용자 정의를 유지하려면 동일한 패치를 다시 제공해야 한다. 이는 동일한 디렉토리로 지정된 `--patches`
```